frontend/src/Components/Login/Login.jsx

```diff
@@ -1,6 +1,138 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 import "./Login.css";
+import logoImage from "./assets/logoblackandwhite.svg";
+import AxiosInstance from "../axios/AxiosInstance";
 
-export const Login = () => {
-  return <div>SkyNest</div>;
+const Login = ({ setAccessToken }) => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [errorMsg, setErrorMsg] = useState("");
+	const [showPassword, setShowPassword] = useState(false);
+	const [loading, setLoading] = useState(false);
```

Comment:
Multiple useState-s can be a good candidate for useReducer hook. Not a must, just a thought.